scipy/spatial/_spherical_voronoi.py

```diff
@@ -238,5 +227,6 @@ def sort_vertices_of_regions(self):
         generator in points and obtain a sorted version of the vertices
         of its surrounding region.
         """
-
+        if self.dim != 3:
+            raise Exception("Only supported for three-dimensional point sets")
```

Comment:
Good suggestion.  I have replaced it with a `TypeError` and added a unit test for it.

Reply:
`ValueError` [may be more appropriate](https://stackoverflow.com/a/48343484/2942522)--perhaps just switch that over the next time you make a more substantial change, since it is a minor point

Reply:
I have made it a `ValueError`.